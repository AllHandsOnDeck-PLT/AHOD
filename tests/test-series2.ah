series<int> a
series<string> b
main:
  {
    a = [2] 
    a.push(1)
  	do PRINT(a[1])
    a.push(3)
    do PRINT(a[2])
    do PRINT(a[2-1])
    b = ["stmt1", "stmt2"]
    b.push("stmt3")
    b.push("stmt4")
    do PRINT(b[3])
  }
  