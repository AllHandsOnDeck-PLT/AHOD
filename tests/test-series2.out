1
3
1
stmt4